cves/kernel/CVE-2020-14386.yml

```diff
@@ -274,8 +300,11 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
```

Comment:
I'm not sure if this is sandboxing since the vulnerability is in the regular networking feature. Can you look into this?

Reply:
Yea, not sure why I marked this as true